### PR TITLE
Add support for IOS

### PR DESCRIPTION
**Pull request by rainyl**
- add support for ios
- arm64(armv8) and x64(x86_64, for simulator) are supported now
- fix dynamiclibrary load in base.dart

### CODE REVIEW

File: ios/opencv_dart.podspec [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  s.author           = { 'Your Company' => 'email@example.com' }
+
+  # This will ensure the source files in Classes/ are included in the native
+  # builds of apps using this FFI plugin. Podspec does not support relative
+  # paths, so Classes contains a forwarder C file that relatively imports
+  # `../src/*` so that the C sources can be shared among all target platforms.
+  s.source           = { :path => '.' }
+  # s.source_files = 'Classes/**/*'
+  s.dependency 'Flutter'
+  s.platform = :ios, '11.0'
```
Comment (azlekov) [inline ios/opencv_dart.podspec:23]: Should this be 12.0 as specified in `profiles/ios-armv8`?

File: ios/opencv_dart.podspec [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+Pod::Spec.new do |s|
+  s.name             = 'opencv_dart'
+  s.version          = '0.0.1'
+  s.summary          = 'A new Flutter FFI plugin project.'
+  s.description      = <<-DESC
+A new Flutter FFI plugin project.
+                       DESC
+  s.homepage         = 'http://example.com'
+  s.license          = { :file => '../LICENSE' }
+  s.author           = { 'Your Company' => 'email@example.com' }
```
Comment (azlekov) [inline ios/opencv_dart.podspec:14]: You can put your email similar to the macOS pod spec?

**Review by azlekov — COMMENTED** [review]
Sorry for bothering, I'm just following your great work, which saved my flutter project a week ago.

Let me know if I can help somehow.

File: ios/opencv_dart.podspec (hunk 1 above)
Comment (rainyl) [reply ios/opencv_dart.podspec:23]: Yes, thanks, I will update it soon.

**Pull request merged**